src/ralph/api/routers/statements.py

```diff
@@ -22,13 +33,29 @@
 )
 
 
-ES_CLIENT = ESDatabase()
+@lru_cache()
+def get_backend() -> BaseDatabase:
+    """Returns a BaseDatabase based on configuration."""
+
+    backend_class = get_class_from_name(RUNSERVER_BACKEND, DATABASE_BACKENDS)
+    options = {}
+    for parameter in signature(backend_class.__init__).parameters.values():
+        if parameter.name == "self":
+            continue
+
+        envvar = f"{ENVVAR_PREFIX}_{backend_class.name}_{parameter.name}".upper()
+        value = config(envvar, None)
+        if value is not None:
```

Comment:
Not sure of what I will write is trivial or not. But are we sure that when a parameter has the None value in the configuration but is requested, it won't fail as it will be missing in `options` value?

Reply:
Good point! Thanks) This part has now changed,  we use pydantic settings for configuration and made all backend init arguments optional (by using defaults set by configuration).
One problem might arise if the user intends to pass `None` explicitly for a cli option value to overwrite a default, however, I'm not sure whether this is permitted in our current implementation.